app/build.gradle

```diff
@@ -82,7 +82,13 @@ dependencies {
     //Butter Knife
     compile "com.jakewharton:butterknife:$rootProject.butterKnifeVersion"
     apt "com.jakewharton:butterknife-compiler:$rootProject.butterKnifeVersion"
+
+    //sweet alert dialog box
+    compile 'cn.pedant.sweetalert:library:1.3'
```

Comment:
We can not use, External Library for Material Dialog.  Use this class https://github.com/openMF/self-service-app/blob/development/app/src/main/java/org/mifos/selfserviceapp/utils/MaterialDialog.java